.github/policies/manageAssignees.yml

```diff
@@ -523,6 +519,19 @@ configuration:
               mentionees:
               - dipti-pai
               - bavneetsingh16
+              - ponatara

```

Comment:
should we move towards exclusively using GH teams instead of individual GH users so that we can streamline permissions management across multiple repos and to account for changes to teams?